wiki/views.py

```diff
@@ -1,11 +1,10 @@
 from django.views.generic.list import ListView
-from .models import Article 
+from .models import Article
 
 class ArticleListView(ListView):
     model = Article
     template_name = "wiki/article_list.html"
     context_object_name = "articles"
 
```

Comment:
also add `def get_context_data` to add "categories" to the context

Reply:
https://docs.djangoproject.com/en/5.2/topics/class-based-views/generic-display/#adding-extra-context